src/input_output/FGOutputFG.cpp

```diff
@@ -146,6 +147,35 @@ FGOutputFG::FGOutputFG(FGFDMExec* fdmex) :
 
 //%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+bool FGOutputFG::Load(Element* el)
+{
+  if (!FGOutputSocket::Load(el)) {
+    return false;
+  }
+
+  // Check if there is a <time> element
+  Element* time_el = el->FindElement("time");
+  if (time_el) {
+    // Check if the attribute "type" is specified and is set to "simulation"
+    if (time_el->HasAttribute("type") && time_el->GetAttributeValue("type") == "simulation") {
+      outputOptions.useSimTime = true;
+    }
+
+    // Check if the attribute "resolution" is specified and set to a valid value
+    if (time_el->HasAttribute("resolution")) {
+      if (time_el->GetAttributeValueAsNumber("resolution") <= 1 &&
+          time_el->GetAttributeValueAsNumber("resolution") >= 1e-9) {
+        outputOptions.timeFactor = 1./time_el->GetAttributeValueAsNumber("resolution");
+      } else {
+        return false;
```

Comment:
Ideally, an error message should be issued when such a condition is failed.

Reply:
Makes sense. How about:
```suggestion
      } else {
        cerr << "Not a valid time resolution (valid range is [1e-9, 1]): " << time_el->GetAttributeValueAsNumber("resolution") << endl;
        return false;
      }  
```